src/com/theironyard/novauc/WarmUp2Test.java

```diff
@@ -0,0 +1,28 @@
+package com.theironyard.novauc;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by octavio on 3/12/17.
+ */
+public class WarmUp2Test {
+    @Before
+    public void setUp() throws Exception {
+
+    }
+
+    @After
+    public void tearDown() throws Exception {
+
+    }
+
+    @Test
+    public void frontTimes() throws Exception {
+
+    }
```

Comment:
Octavio you don't need the @Before and @After annotations just the @Test.  Those look to be setup correctly however you need to add logic inside for the test cases.  You need to create an object in the test class to call the method.  So WarmUp2 wt = new WarmUp2();.  Then inside the test method Something like assertEquals(wt.frontTimes("Abc", 3), "AbcAbcAbc").  CodingBat gives you ample test cases so you can just use the examples they give you.  Add that code and resubmit.